docs/sdks/debugging.mdx

```diff
@@ -57,6 +57,22 @@ For server SDKs, these reasons for the value you are seeing can be:
 - `Unrecognized`: the sdk was initialized, but this config did not exist in the set of values
 - `Uninitialized`: the sdk was not yet successfully initialized
 
+In newer server SDKs, the reasons will be a combination of initialization source and evaluation reason:
+Initialization Source
+- `Network`
+- `Bootstrap`
+- `DataAdapter`
+- `Uninitialized`
+- `StatsigNetwork`: this refers to when custom proxy/grpc streaming has triggered the fallback behavior, thus falling back to statsig api. 
+   * If no network config/overrides were used, the sdk default uses statsig apis but the reason will be Network.
+
+Evaluation Reason
+- `LocalOverride`
+- `Unrecognized`
+- `Unsupported`
```

Comment:
either add a definition for each of them, or none of them

Reply:
the definition for the rest of them are above, i only added for the new ones we introduced in v2. I re-listed them these enums have been shifted to reason vs source